sympy/physics/mechanics/loads.py

```diff
@@ -0,0 +1,187 @@
+from abc import ABC, abstractmethod
+from collections import namedtuple
+from sympy.physics.mechanics.body_base import BodyBase
+from sympy.physics.vector import Vector, ReferenceFrame, Point
+
+__all__ = ['LoadBase', 'Force', 'Torque']
+
+
+class LoadBase(ABC, namedtuple('LoadBase', ['location', 'vector'])):
```

Comment:
Another option I was thinking about was to make the `location` and `vector` attributes protected, i.e. `_location`, `_vector`, because the child classes have aliases for them. However from the perspective of parsing the loads it is really nice to have a common name for it, which each load type shares.

Reply:
Could argue either way. But I think having a consistent name across all different load types is potentially more useful.